lib/mayaUsd/ufe/EditAsMayaCommand.h

```diff
@@ -19,6 +19,8 @@
 #include <mayaUsd/base/api.h>
 #include <mayaUsd/undo/OpUndoItemList.h>
 
+#include <usdUfe/ufe/UfeVersionCompat.h>
```

Comment:
Sorry after running preflight I realized many headers were missing the Ufe define UFE_V4